Semantics.md

```diff
@@ -326,7 +326,7 @@ In the MVP, the primary use case of global variables is to represent
 instantiation-time immutable values as a useful building block for
 [dynamic linking](DynamicLinking.md).
 
-After the MVP, when [reference types](GC.md) are added to the set of [value types](#types),
+After the MVP, when [reference types][future garbage collection] are added to the set of [value types](#types),
```

Comment:
This should point to the reference types proposal as this was split out from the GC proposal after this text was added.
https://github.com/WebAssembly/reference-types

Reply:
Updated! Thanks!